app/api/test/mytests.js

```diff
@@ -85,8 +82,33 @@ describe("Test API endpoints", () => {
     it("Should place an order for meal on POST /api/v1/order", done => {
       request(app)
         .get("/api/v1/order")
+        .expect(200, done);
+    });
+  });
+
+  describe("Test users endpoint", () => {
+    it("Should add a new user at POST /auth/signup", done => {
+      request(app)
+        .post("/auth/signup")
+        .send({
+          firstName: "Micah",
+          lastName: "Bala",
+          userName: "micahbala",
+          password: "micahbala",
+          isAdmin: false
+        })
```

Comment:
You want to add robust assertions to your test cases. Your expect method should assert that the object created here has all the necessary fields (firstName, lastName, etc) in them